pkg/static/login.js

```diff
@@ -340,8 +340,12 @@
         window.onload = null;
 
         translate();
-        if (window.cockpit_po && window.cockpit_po[""])
-            document.documentElement.lang = window.cockpit_po[""].language || "en-us";
+        if (window.cockpit_po && window.cockpit_po[""]) {
+            const language = window.cockpit_po[""].language || "en-us";
+            const lang_dir = language.match(/he-\*/) ? 'rtl' : 'ltr';
```

Comment:
I get that the login page needs special treatment/duplication, as we have neither shell nor cockpit.js. But it should use the same test. This is not just different, it also looks wrong -- `\*` machtes a literal asterisk, which is certainly not applying to any language?